### ISSUE TITLE
Silent crash on Ubuntu MATE and Xubuntu 15.10

### ISSUE THREAD

**Issue by nikolay** [atom/atom#9933]
Both the stable and the beta release crash silently on Ubuntu 15.10 using both MATE and Xubuntu desktops. There's nothing in the system logs and even launching with `--safe` or `-f -d` or `-l` options make no difference. Any idea what's the problem?

**Comment by lee-dohm** [comment]
Can you give the version numbers of the stable and beta releases that you have installed? This can help us narrow things down.

**Comment by nikolay** [comment]
The stable version is 1.2.4 and the beta is 1.3.0-beta6. I tried both your `.deb` packages and also the PPA `webupd8team/atom` - both behave the same and silently crash.

**Comment by nikolay** [comment]
Continues to crash even on 16.04.

**Comment by leefr** [comment]
Not sure if this is related -- I am seeing v 1.2.4 freeze while sitting idle with 2 files open in a vertical split screen under MATE on Ubuntu 14.04.  Just opening the one file file, splitting the screen (right), opening the second, scrolling around for a few seconds then leaving idle for a minute or so -- Atom has become unresponsive when I come back to it.  Closing the window causes a pause then the system dialog that the app is unresponsive and picking the Close button is the only way to get out.

I was able to repeat this behavior with the above sequence twice.

**Comment by Zireael07** [comment]
1.2.4? We're at 1.6.something now!

**Comment by leefr** [comment]
OK -- same behavior on 1.6.1 (just updated when I realized I had disabled the PPA when looking into some mirroring issues a while back).  So obviously it's a long standing issue that has not been addressed. There was a slight difference on 1.6.1 in that I was notified that the editor was unresponsive without trying to do anything and that notification dialog had popped up multiple times during the during Atom's "idle" (no active interaction from me) period.
